contract/src/lib.rs

```diff
@@ -232,12 +266,12 @@ impl BtcLightClient {
         let initial_blockheader = self
             .headers_pool
             .get(&self.mainchain_initial_blockhash)
-            .unwrap_or_else(|| env::panic_str("initial blockheader must be in a header pool"));
+            .unwrap_or_else(|| env::panic_str(ERR_KEY_NOT_EXIST));
```

Comment:
Not sure we should be switching to less explicit error message unless there is a good reason?

Reply:
It is to reduce the contract size, but it is okay to have a one general error message like this for all impossible scenarios, so it is like `.unwrap` 
See this https://github.com/near/near-sdk-rs/blob/0be2c7109bff71d0fac7b37b1463b980b4462568/near-sdk/src/store/mod.rs#L110